server/src/main/java/com/senlainc/warsaw/tyurin/service/CraftsmanService.java

```diff
@@ -10,20 +12,27 @@
 import com.senlainc.warsaw.tyurin.util.jsonHandlers.JsonReader;
 import com.senlainc.warsaw.tyurin.util.jsonHandlers.JsonWriter;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 import java.util.stream.Collectors;
 
+@DependencyClass
 public class CraftsmanService implements ICraftsmanService{
 
     private static CraftsmanService INSTANCE;
+    @DependencyComponent
     private ICraftsmanDAO craftsmanDAO;
+    @DependencyComponent
     private IOrderService orderService;
     private CsvReader csvReader;
```

Comment:
Why readers and writers not injected through DI? They are also application components and must be injected like services and repos